src/ken/src/ken_server.erl

```diff
@@ -267,22 +267,26 @@ get_active_count() ->
 
 % If any indexing job fails, resubmit requests for all indexes.
 update_db_indexes(Name, State) ->
-    {ok, DDocs} = design_docs(Name),
```

Comment:
The logged error was:
```
{{badmatch,{error,timeout}},[{ken_server,update_db_indexes,2,[{file,"src/ken_server.erl"},{line,278}]}]}
```
but with this diff the `badmatch` would presumably occur at line 270. I'm guessing they were using an older version?

Reply:
It's because the workload is using the 3.3.2 version. There it should match up to line 278 https://github.com/apache/couchdb/blob/11a2340709af2e3ebcb052337c9acb88f7fac3cb/src/ken/src/ken_server.erl#L278